src/components/pacbio/PacbioTagEditModal.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <b-button :id="'editTag'+this.tag.id" size="sm" @click="show" variant="outline-primary">Edit</b-button>
+
+    <b-modal
+      id="editTagModal"
+      ref="modal"
+      title="Edit Tag"      
+    >    
+
+      <b-form id="editTagForm" v-on:submit.prevent>
```

Comment:
What does submit.prevent do?

Reply:
not needed in this instance, I will remove it, it is a vue method with stops a form being submitted